drivers/sony/displays/bravia.cr

```diff
@@ -0,0 +1,269 @@
+require "placeos-driver/interface/powerable"
+require "placeos-driver/interface/muteable"
+require "placeos-driver/interface/switchable"
+
+class Sony::Displays::Bravia < PlaceOS::Driver
+  include Interface::Powerable
+  include Interface::Muteable
+
+  INDICATOR = "\x2A\x53" # *S
+  HASH      = "################"
```

Comment:
These can both move into `#do_send` inline as part of the serialisation.

Reply:
i think these are fine, just make them private constants so they don't show up in documentation